purchase_order_line_display_sequence/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* Cécile Jallais <cjallais@archeti.com>
+* Damien Crier <damien.crier@camptocamp.com>
+* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
```

Comment:
Can you please change 
* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
as below
* ForgeFlow S.L. <contact@forgeflow.com>

otherwise looks good